atom/common/crash_reporter/crash_reporter_win.cc

```diff
@@ -14,6 +14,7 @@
 #include "content/public/common/result_codes.h"
 #include "gin/public/debug.h"
 #include "sandbox/win/src/nt_internals.h"
+#include "base/command_line.h"
```

Comment:
Not sure if I am a big fan of adding extra command line dependency here, couldn't you use the extra parameters provided in crash_reporter.cc and depending on existence or not of one do the same logic?